packages/shex-webapp/doc/shex-simple.js

```diff
@@ -1719,7 +1720,15 @@ function loadSearchParameters () {
   });
 }
 
-  /**
+function changeInputTabs() {
```

Comment:
I think this name is a bit too generic – it should explain how it changes the input tabs :) perhaps `changeInputTabsForSparqlQueryMap()`?

(I’d also be fine with inlining the method directly into `loadSearchParameters()`, actually, since it’s not very long.)